lms/djangoapps/teams/static/teams/js/spec/views/team_profile_spec.js

```diff
@@ -119,10 +119,6 @@ define([
                     assertTeamDetails(view, 0, false);
                     expect(view.$('.team-user-membership-status').length).toBe(0);
 
-                    // Verify that invite and leave team sections are not present.
-                    expect(view.$('.leave-team').length).toBe(0);
```

Comment:
I noticed while rebasing that you accidentally took out the leave-team tests. I'm adding them back in my PR. Sorry that I didn't notice this!